CONDUCT.md

```diff
@@ -0,0 +1,98 @@
+# Scikit-Optimize Code of Conduct
+
+Scikit-Optimize is an engaged and respectful community made up of people
+from all over the world. Your involvement helps us to further our
```

Comment:
Here is a thought: how about adding a few sentences in the beginning, which state that in the end breaking the code is not good for the offender themselves? My expectation is when it is focused on benefit of the person that reads the code this will be more effective


"
The goal of scikit-optimize is to become the best software for black box optimization, freely available to everyone. As any other open source software, it requires a community of people who maintain and contributes to it. The quality of the software depends on the engagement and size of the community; thus you as a user of skopt directly benefit from creating positive environment that encourages people to contribute to the software.

To ensure that we build the best possible environment for our community, we introduce this code of conduct. On the one hand side, when these guidelines are not followed, it can be detrimental to the community, and in turn not beneficial to the reader, who is expected to utilize skopt. On the other hand side, as all work is public on github, this exists to ensure that no possible negative effect for the reputation of the person who might break the code is done. 

"

Reply:
I like it! I used most of it: I tweaked the first paragraph replacing "software" with "project". Edited the second paragraph a bit before adding (made a new annotation to discuss it)

Reply:
:+1: